modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,63 @@
+= Share an Asset
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+:page-aliases: to-share-an-asset-with-a-user.adoc
+
+Exchange administrators can share an asset with another user in their organization, with the whole organization, or with an external organization.
+
+== Share With a User
+
+If the person is not in your organization,
+click Access Management > Users to add them.
+
+. Click an asset to show its details and click Share.
+. In the Share menu, type a few letters of the person's name you want to share this asset with.
+Or click a filter to list only users who have Viewer, Contributor, or Admin rights.
+. Click Add.
+. Repeat steps 2 and 3 to add additional users to share the asset with.
+
+You can also use the Share menu to change who you are sharing the asset with or to stop sharing with one or more users.
+
+== Share With Your Organization
```

Comment:
Recently news, That feature won't be in the release, please removed it

Reply:
OK, I removed "Share With Your Organization". Thank you for telling me.